.github/workflows/__release-workflow.yaml

```
@@ -71,7 +71,7 @@ on:
         default: false
 
 permissions:
-  contents: read
+  contents: write
```

This also shouldn't be required. The only workflow that we call here is `.github/workflows/__build-workflow.yaml` which doesn't require any additional permissions.


```suggestion
  contents: read
```